WooCommerce/WooCommerceTests/Authentication/StoreCreationCoordinatorTests.swift

```diff
@@ -62,4 +68,51 @@ final class StoreCreationCoordinatorTests: XCTestCase {
         let storeCreationNavigationController = try XCTUnwrap(navigationController.presentedViewController as? UINavigationController)
         assertThat(storeCreationNavigationController.topViewController, isAnInstanceOf: AuthenticatedWebViewController.self)
     }
+
+    // MARK: - Presentation in different states for store creation M2
+
+    func test_AuthenticatedWebViewController_is_presented_when_navigationController_is_presenting_another_view() throws {
```

Comment:
Nit: I think these test methods should have `AuthenticatedWebViewController` in their name. 😄

Reply:
Good catch! Updated in https://github.com/woocommerce/woocommerce-ios/pull/8065/commits/15e1319d652c3a2e552e20ed4a5a0d773ef05b5d